examples/default.yaml

```diff
@@ -61,11 +61,28 @@ mounts:
     # system will look and feel like regular files directories in the Guest OS.
     # 🟢 Builtin default: false
     followSymlinks: null
+  9p:
+    # Supported security models are "passthrough", "mapped-xattr", "mapped-file" and "none".
+    # 🟢 Builtin default: "mapped-xattr"
+    securityModel: null
+    # Select 9P protocol version. Valid options are: "9p2000" (legacy), "9p2000.u", "9p2000.L".
+    # 🟢 Builtin default: "9p2000.L"
+    protocolVersion: null
+    # The number of bytes to use for 9p packet payload, where 4KiB is the absolute minimum.
+    # 🟢 Builtin default: "128KiB"
+    msize: null
+    # Specifies a caching policy. Valid options are: "none", "loose", "fscache" and "mmap".
+    # 🟢 Builtin default: "mmap"
```

Comment:
Let me change this to `fscache`
- https://github.com/lima-vm/lima/issues/786

Reply:
That is a breaking change, though.

Reply:
I like the original comments on the 9p file system, it had some reflections on cache.

https://landley.net/kdocs/Documentation/filesystems/9p.txt